openshift/job-template.yaml

```diff
@@ -111,3 +111,23 @@ parameters:
     required: true
     name: IMAGE_STREAM_TAG
     value: 'latest'
+  - description: Registry the ImageStream to be use lives in
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+  - description: Project the ImageStream to be use lives in
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
+  - description: Tag of the ImageStream to be use
```

Comment:
there is a typo in ```use``` here and also in the descriptions above :) otherwise LGTM. thanks :1st_place_medal: